hydra-node/src/Hydra/Network/Ouroboros.hs

```diff
@@ -254,9 +256,14 @@ withOuroborosNetwork tracer localHost remoteHosts networkCallback between = do
       , recvMsgDone = pure ()
       }
 
-data TraceOuroborosNetwork
+data WithHost trace = WithHost Host trace
```

Comment:
Isn't hlint warning that this could be a `newtype`? if yes, I'd follow that suggestion